GitUI/UserControls/FileStatusList.Designer.cs

```diff
@@ -151,18 +150,59 @@ private void InitializeComponent()
             DeleteFilterButton.Location = new Point(646, 1);
             DeleteFilterButton.Name = "DeleteFilterButton";
             DeleteFilterButton.Size = new Size(18, 19);
-            DeleteFilterButton.TabIndex = 1;
             DeleteFilterButton.Click += DeleteFilterButton_Click;
             // 
+            // SearchComboBox
+            // 
+            SearchComboBox.Dock = DockStyle.Top;
+            SearchComboBox.FlatStyle = FlatStyle.Flat;
+            SearchComboBox.FormattingEnabled = true;
+            SearchComboBox.Location = new Point(0, 23);
+            SearchComboBox.Margin = new Padding(0, 0, 0, 1);
+            SearchComboBox.Name = "SearchComboBox";
+            SearchComboBox.Size = new Size(682, 21);
+            SearchComboBox.Tag = "ToolBar_group:Text search";
+            SearchComboBox.TabIndex = 2;
+            SearchComboBox.SelectedIndexChanged += SearchComboBox_SelectedIndexChanged;
+            SearchComboBox.SizeChanged += SearchComboBox_SizeChanged;
+            SearchComboBox.GotFocus += SearchComboBox_GotFocus;
+            SearchComboBox.LostFocus += SearchComboBox_LostFocus;
+            SearchComboBox.TextUpdate += SearchComboBox_TextUpdate;
+            // 
+            // SearchWatermarkLabel
+            // 
+            SearchWatermarkLabel.AutoSize = true;
+            SearchWatermarkLabel.BackColor = SystemColors.Window;
+            SearchWatermarkLabel.ForeColor = SystemColors.GrayText;
+            SearchWatermarkLabel.Location = new Point(4, 27);
+            SearchWatermarkLabel.Name = "SearchWatermarkLabel";
+            SearchWatermarkLabel.Size = new Size(184, 13);
+            SearchWatermarkLabel.Text = "Search files in revision using a regular expression...";
```

Comment:
❔
```suggestion
            SearchWatermarkLabel.Text = "Search text in all files of revision using a regular expression...";
```
might be clearer